src/gui/EditWidgetIcons.cpp

```diff
@@ -65,6 +69,9 @@ EditWidgetIcons::EditWidgetIcons(QWidget* parent)
     connect(m_ui->faviconButton, SIGNAL(clicked()), SLOT(downloadFavicon()));
 
     m_ui->faviconButton->setVisible(false);
+
+    m_fallbackToGoogle = true;
+    m_redirectCount = 0;
```

Comment:
These are only basic types, so generally not a big deal, but this should either be initialized in the header or with an initialization list.